discogs-labels.csv

```diff
@@ -300,3 +300,5 @@ Label ID, Label Name, Link
 2466, Flying Records, https://www.discogs.com/label/2466-Flying-Records
 48, Wave Music, https://www.discogs.com/label/48-Wave-Music
 8574, Muve Recordings, https://www.discogs.com/label/8574-Muve-Recordings
+IbogaRecordsMusic, Iboga Records Music, https://www.youtube.com/c/IbogaRecordsMusic
+@SpinTwistRecords, SpinTwistRecords, https://www.youtube.com/@SpinTwistRecords
```

Comment:
you are in the discogs file, you want to use the youtube file

Reply:
yeah my bad